packages/contracts-bedrock/src/Safe/GetSigners.sol

```diff
@@ -0,0 +1,88 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+library GetSigners {
```

Comment:
Thoughts on just naming this `Signers` or `SafeSigners`?

Reply:
This was resolved without being changed

Reply:
8994c68b7b904468c959a41db2a0adfffab1d5ef